style.css

```diff
@@ -0,0 +1,128 @@
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+html {
+  font-family: 'Roboto', sans-serif;

```

Comment:
[Optional] Please note that the Figma template uses `Poppins` font-family and not `Roboto`, for the design to be consistent with the Figma template please change the font-family to `Poppins`